main.go

```diff
@@ -19,9 +19,56 @@ limitations under the License.
 package main
 
 import (
-	"fmt"
+	"flag"
+	"os"
+
+	"sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/controllers"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/api/v1alpha1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	// +kubebuilder:scaffold:imports
```

Comment:
What's this for?

Reply:
kubebuilder uses these scaffold markers as a hook to add more things during a scaffold create action (such as making a new crd)

Reply:
Interesting thanks!